theano/gof/opt.py

```diff
@@ -1131,13 +1131,18 @@ class LocalMetaOptimizer(LocalOptimizer):
 
     """
 
-    def __init__(self, tracks=None, optimizers=()):
-        self._tracks = tracks
-        self.optimizers = list(optimizers)
+    def __init__(self):
         self.verbose = config.metaopt.verbose
```

Comment:
You still need to create an empty `_tracks` list otherwise the append in register() will crash.

You also need an empty `optimizers` list for the same reason.